### DISCUSSION TITLE
account compromised

### DISCUSSION THREAD

**Discussion by marschallarkan** [community/community#113300] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I would like to return my account or delete it What do I have to do?

**Comment by ettaboyle** [comment]
Hi @marschallarkan! Thanks for sharing this. If your account is in fact compromised, you'll need to open a ticket on our [Support page](https://support.github.com/contact). We cannot answer or respond to account specific questions in this public forum. 

If you can still access your account _(if its the one you posted this with, i'm assuming yes)_ you can find [instructions on how to delete your personal account](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/deleting-your-personal-account#deleting-your-personal-account) in the [GitHub Docs](https://docs.github.com/en). You can also find answers to questions like this with a quick browser search ;) 

Thanks for being a part of the GitHub community!
